README.md

```diff
@@ -31,6 +31,12 @@ export INSTALL_MOD_PATH=$TARGET_DIR
 It should install the mali.ko Linux kernel module into the target filesystem,
 and the module should be loaded automatically. If it isn't, modprobe will help.
 
+To compile for arm64(aarch64) platform, before issuing ./build.sh :
+
+```
+export ARCH=arm64
+```
```

Comment:
exporting that variable has other issues. Why not stating that you need to set the ARCH environment variable, leaving the user use whatever method they seem fit?

This should probably be earlier in the documentation too, right next to when you compile

Reply:
Yes. Ok I modify it.
Thanks for pointing.